scipy/stats/mstats_basic.py

```diff
@@ -526,6 +526,47 @@ def _spearmanr_2cols(x):
         return SpearmanrResult(rs, prob)
 
 
+def _kendall_p_exact(n, c):
+    # Exact p-value, see Maurice G. Kendall, "Rank Correlation Methods" (4th Edition), Charles Griffin & Co., 1970.
+    if n <= 0:
+        raise ValueError
+    elif c < 0 or 4*c > n*(n-1):
+        raise ValueError
+    elif n == 1:
+        prob = 1.0
+    elif n == 2:
+        prob = 1.0
+    elif c == 0:
+        prob = 2.0/np.math.factorial(n)
+    elif c == 1:
+        prob = 2.0/np.math.factorial(n-1)
+    elif 4*c == n*(n-1):
+        prob = 1.0
+    elif n < 171:
+        new = np.zeros(c+1)
+        new[0] = 1.0
+        new[1] = 1.0
+        for j in range(3,n+1):
+            cs = np.cumsum(new)
+            new = cs.copy()
+            if j <= c:
+                new[j:] -= cs[:c+1-j]
+        prob = 2.0*sum(new)/np.math.factorial(n)
```

Comment:
Do you want `sum(new)` or `np.sum(new)`?

Reply:
Both work, so switched to `np.sum(new)`.